IdentityCore/src/MSIDConstants.m

```diff
@@ -49,3 +49,6 @@
 
 NSString *const MSID_DEFAULT_AAD_AUTHORITY         = @"https://login.microsoftonline.com/common";
 NSString *const MSID_DEFAULT_MSA_TENANTID          = @"9188040d-6c67-4c5b-b112-36a304b66dad";
+
+NSString *const MSID_CLIENT_SDK_TYPE_MSAL         = @"sdk_msal";
```

Comment:
are we going to pass this string to ESTS? If yes, can we make it standard with what we have been using before?

Reply:
no, we don't pass it to ests. It is being used for SSO ext protocol only.